app/controllers/photos_controller.rb

```diff
@@ -8,6 +8,7 @@ def find_album
 
   def find_photo
     @photo = Photo.find(params[:id])
+    @photo.view += 1;
```

Comment:
Also, do you want the view to be incremented each time the photo is found? I think sometimes the photo object might be found but not necessarily used (like if you wanted to link it or only display a thumbnail, something like that).